src/unittest/python/tracker_tests.py

```diff
@@ -73,6 +74,59 @@ def test_point_not_in_area(self):
         self.assertEqual(len(self.mock_timer_instance.clear.mock_calls), 2)
 
 
+class TestTrackerCalibrate(TestCase):
+
+    def setUp(self):
+        self.mock_queue = mock.patch('vyu.tracker.Queue').start()
+        self.mock_process = mock.patch('vyu.tracker.Process').start()
+        self.mock_get_reader = mock.patch(
+            'vyu.tracker.imageio.get_reader').start()
+        self.mock_estimate_matrices = mock.patch(
+            'vyu.tracker.calibration.estimate_matrices').start()
+        self.mock_estimate_matrices.return_value = ('ANY_MATRIX', 'ANY_VECTOR')
```

Comment:
I had a look at the `unittest` documentation but its still unclear to me what the point of `start()` is. The documentation mentions it's a way to eliminate the use of decorators, though, as the name would  suggests, it has no mention of initialization of these `mock.patch()` objects

Reply:
Using explicit calls to `mock.patch.start` here. See documentation.